mods/ra/rules/structures.yaml

```diff
@@ -51,7 +51,6 @@ MSLO:
 		DisplayBeacon: True
 		DisplayRadarPing: True
 		BeaconPoster: atomicon
-		FlashType: Nuke
```

Comment:
I just noticed this change is in a different commit than it's C# counterpart, which means things would probably break if someone was to jump in the middle of the commits here, say during a bisect. I think we usually ignore such things, and I want to ask this as more general question: @reaperrr, @pchote do we want to establish any sort of ruling for how such things should be handled? (a. Ignore as before, since it is unlikely to have an impact and is easily worked around if it has, or b. force commits to always compile and run the game)

Reply:
I don't think it's feasible to hope every single commit to be stable and working, as a good chunk of them will be parts of major refactorings these days.

Reply:
The logic doesn't need to work perfectly, but it would be very helpful and should be possible to have each commit compile and at least start the game.